Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:slim
+
+RUN apt update && apt install -y git
+
+WORKDIR /usr/src/jslt
+
+COPY . .
+
+RUN ./gradlew :playground:shadowJar
```

Comment:
I suggest switching to a multi-stage build, so that the final docker container has only the JSLT jar on top of a slimmer JRE image. As it stands the final container will contain the full JDK plus any intermediate build artifacts.

Another alternative would be to build the jar outside docker and then package it in a vanilla JRE image directly. This has the downside that the Dockerfile and a running docker setup are no longer self-sufficient for bootstrapping.

Reply:
This sounds very reasonable to me.

The number of dependencies is very low. Perhaps we could just get them from Maven Central with `wget` or something?

```
+--- project :core
|    \--- com.fasterxml.jackson.core:jackson-databind:2.12.0
|         +--- com.fasterxml.jackson.core:jackson-annotations:2.12.0
|         \--- com.fasterxml.jackson.core:jackson-core:2.12.0
\--- org.eclipse.jetty:jetty-server:9.4.29.v20200521
     +--- javax.servlet:javax.servlet-api:3.1.0
     +--- org.eclipse.jetty:jetty-http:9.4.29.v20200521
     |    +--- org.eclipse.jetty:jetty-util:9.4.29.v20200521
     |    \--- org.eclipse.jetty:jetty-io:9.4.29.v20200521
     |         \--- org.eclipse.jetty:jetty-util:9.4.29.v20200521
     \--- org.eclipse.jetty:jetty-io:9.4.29.v20200521 (*)
```

We could also host the shadowJar as a github release or something, but this seems maybe simpler and more straightforward?